lib/utils/release-utils.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type * as BalenaSdk from 'balena-sdk';
+
+export async function getRelease(
```

Comment:
Note:  `balena.models.release.get()` isn't smart enough if you pass a number as a string (`id`) to do the right thing.  In such a case, you always get "not found".  So we have to try to see if we can parse it as a number and if we can, attempt first to get it as an id.  In most cases, the `intParse` will fail, so we won't actually get multiple attempts.